### ISSUE TITLE
Audio player doesn't have a seek bar if the visualization doesn't load or at times where there's silence

### ISSUE THREAD

**Issue by noiob** [tootsuite/mastodon#12277]
### Expected behaviour

The audio player embed should have a usable seek bar if the waveform visualization doesn't generate or if there's silence

### Actual behaviour

The audio player embed just has a hairline and no indication of where in the track you actually are
![grafik](https://user-images.githubusercontent.com/8197071/68056091-9266b300-fcf2-11e9-99e9-322c1a834ca2.png)
(this is https://mastodon.social/@Gargron/103064569807259308 but with CORS misconfigured so there's no way for the frontend to generate the waveform)

### Steps to reproduce the problem

- upload an audio file with silent parts or mess up your CORS configuration so the waveform can't generate
- play said audio file

### Specifications

v3.0.1+glitch

**Comment by skobkin** [comment]
> https://mastodon.social/@Gargron/103064569807259308 but with CORS misconfigured so there's no way for the frontend to generate the waveform

It's interesting. I have probably same (waveform generation) problem with my post:
https://lor.sh/@skobkin/103062340004571180

I thought that it's server problem. But then I tested it in Firefox (<70) and it worked when Vivaldi couldn't show the waveform. Then I updated my Firefox and it's now can't draw it too.

But more interesting that I can see waveform in the post you linked in both browsers.

**Comment by MaciekBaron** [comment]
I think a quick solution could be adding a faint background so that even if it's silent you can still see it.

![image](https://user-images.githubusercontent.com/1668712/68545099-97e28e00-03c1-11ea-8788-d48047c3ecad.png)

**Comment by nightpool** [comment]
@MaciekBaron I like that solution, although i'm not sure I understand how it's an accessibility problem—it just seems like an oversight in handling silent parts of the waveform?

Another option would just be to make "silent" hairline a little wider, and have it turn blue as play proceeds along it, just like the waveform does.

**Comment by noiob** [comment]
Adding to this that it takes a considerable amount of time to download the file and create the waveform and until then the entire seek bar area looks weird and doesn't give any visual feedback

**Comment by noiob** [comment]
closing this since the wavesurfer player has been deprecated anyways
